gcp_variant_transforms/vcf_to_bq_preprocess.py

```diff
@@ -87,6 +91,21 @@ def _get_inferred_headers(variants,  # type: pvalue.PCollection
   return inferred_headers, merged_header
 
 
+# TODO(hanjohn): Add an e2e test
+def _estimate_disk_resources(p, input_pattern):
+  # type: (pvalue.PCollection, str) -> (pvalue.PCollection)
+  # TODO(hanjohn): Add support for `ReadAll` pattern for inputs with very large
+  # numbers of files.
+  result = (
+      p
+      | 'InputFilePattern' >> beam.Create([input_pattern])
+      | 'ReadFileSizeAndSampleVariants' >> vcf_file_size_io.EstimateVcfSize(
+          input_pattern, _SNIPPET_READ_SIZE)
+      | 'SumFileSizeEstimates' >> beam.CombineGlobally(
```

Comment:
How about moving this combine into EstimateVcfSize PTransform? Would we ever call EstimateVcfSize without summing the results?

Reply:
sg; also inlined the rest of the code directly in run() since it's just a single pipe now